quantum-mechanics/README.md

```diff
@@ -1,3 +1,12 @@
-## Quantum Mechanics 
+# Quantum Mechanics 
 
-This course is very different from all the other courses mentioned above, for it focuses on delivering a scientifically pure approach towards Quantum Mechanics which can later be manifested and utilized in understanding and working on Quantum Computing. This course covers everything from the very grass-root level of concepts and helps develop a pretty concrete understanding and intuition for almost all aspects of modern physics backed by Quantum Mechanics. It is presented by MIT OCW and the instructor in charge is Dr. Barton Zweibach.
+This discipline of science is quite different and even nascent with regards to the lab. This is the stepping stone to an entire flight of steps towards almost every modern science, right from Quantum Chromodynamics to Quantum Computing.
+
+
+## Courses
+### [Quantum Physics I](https://ocw.mit.edu/courses/physics/8-04-quantum-physics-i-spring-2016/)
+This course is very different from all the other courses, for it focuses on delivering a scientifically pure approach towards Quantum Mechanics which can later be manifested and utilized in understanding and working on Quantum Computing. This course covers everything from the very grass-root level of concepts and helps develop a pretty concrete understanding and intuition for almost all aspects of modern physics backed by Quantum Mechanics. It is presented by MIT OCW and the instructor in charge is Dr. Barton Zweibach.
```

Comment:
all other courses? Since this is an independent README file, just start with "This course focuses on delivering a scientifically pure approach ..................."